src/bio/smd/smd_device.c

```diff
@@ -198,8 +198,9 @@ fetch_dev_info(uuid_t dev_id, struct smd_dev_info **dev_info)
 	rc = dbtree_fetch(smd_store.ss_dev_hdl, BTR_PROBE_EQ,
 			  DAOS_INTENT_DEFAULT, &key, NULL, &val);
 	if (rc) {
-		D_ERROR("Fetch dev "DF_UUID" failed. %d\n",
-			DP_UUID(&key_dev.uuid), rc);
+		D_CDEBUG(rc != -DER_NONEXIST, DLOG_ERR, DB_MGMT,
```

Comment:
What does D_CDBEUG do? This doesn’t seem like is going to pollute debug log, might be better to leave it as D_ERROR as it is a fatal error if SMD fetches fail?

Reply:
It allows you to specify multiple masks in a single debug statement based on the condition

So if the condition here is true, it will use DLOG_ERR while it will use DB_MGMT if it's false.